hack/relprep.sh

```
@@ -76,17 +67,19 @@ echo "Using a copy of previous release version $prev as a starting point for $ve
 # using tar for easy overwrite of existing files (for idempotency, in case the script gets runs more than once)
 mkdir -p keda/$ver
 tar cf - --directory keda/$prev . | tar xvf - --directory keda/$ver
+rm -f keda/${ver}/manifests/cma.v*.clusterserviceversion.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)